### ISSUE TITLE
osrm 4.7.1

### ISSUE THREAD

**Issue by freenerd** [Project-OSRM/osrm-backend#1598]
The develop branched has [diverged quite a lot](https://github.com/Project-OSRM/osrm-backend/compare/master...develop) from master. Wouldn't it be time for a new release, at least 4.7.1? There is [a milestone](https://github.com/Project-OSRM/osrm-backend/milestones/4.7.1) but not sure if it really covers the work TBD.

@TheMarex What's the chance of releasing a new version on Monday?

**Comment by TheMarex** [comment]
The original 4.7.1 milestone was set quite a while ago and got outdated as it became clear that priorities shifted. Need to read the backlog a bit.

**Comment by TheMarex** [comment]
I'm going to push out `4.7.1` today containing the following commits:

```
9958937fd1c1f9dd60126a56e1c4f25ceefaf70e At least check 4*LEAF_SIZE edges before returning none.
f19c57200da0f9c79cdda5c2bab274df09be79ac Fix endless loop
8a2652f53db1237085030fd4c517fe94b244d001 Only penaltize bidirectional ways if they have 1 lane
dddde4ddabb5dc9f2f67b5b6319aca0c20690460 Fix backwards speed on oneway=-1 streets
```
Should be fairly painless release.

**Comment by TheMarex** [comment]
- [x] Tag `osrm-backend` release
- [x] Tag `node-osrm` release
- [ ] Publish announcement to ML:

```
This is a quick bug-fix release 4.7.1 containing four important fixes:

- The coordinate lookup will now better find roads in sparse areas
- An endless loop for broken input data was fixed
- Only bidirectional ways are now penaltized if they have 1 lane
- Backwards speed on oneway=-1 streets are now set correctly

The 4.8.0 release will probably follow at the end of this month.
```

**Comment by TheMarex** [comment]
All done here.
